dashboards-observability/public/components/visualizations/charts/maps/heatmap.tsx

```diff
@@ -37,7 +37,9 @@ export const HeatMap = ({ visualizations, layout, config }: any) => {
     isEmpty(zMetrics) ||
     isEmpty(data[xaxisField.label]) ||
     isEmpty(data[yaxisField.label]) ||
-    isEmpty(data[`${zMetrics.aggregation}(${zMetrics.name})`])
+    isEmpty(data[`${zMetrics.aggregation}(${zMetrics.name})`]) ||
+    dataConfig?.dimensions.length > 2 ||
```

Comment:
what if only 1 dimension is selected?

Reply:
If less than 2 dimension are selected the Heatmap doesn't render. That case was already handled.

Reply:
ok cool, any hints on UI to show to the user?

Reply:
<img width="955" alt="image" src="https://user-images.githubusercontent.com/61182817/191173846-71ba938e-6d72-4df5-8765-a85fd8b45447.png">

This is how it currently shows in the UI